doc/classes/SurfaceTool.xml

```diff
@@ -123,6 +123,8 @@
 			<argument index="0" name="nd_threshold" type="float" />
 			<argument index="1" name="target_index_count" type="int" default="3" />
 			<description>
+				Generates a LOD for a given [code]nd_threshold[/code] in linear units (square root of quadric error metric), using at most [code]target_index_count[/code] indices.
+				Deprecated. Unused internally and neglects to preserve normals or UVs. Consider using [method ImporterMesh.generate_lods] instead.
```

Comment:
Out of scope for this PR, but we should really consider either removing this function, or copying the implementation from ``generate_lods`` and then using this one in the ImporterMesh